src/app/ng-intercom/intercom/intercom.ts

```diff
@@ -1,5 +1,6 @@
 import { Inject, Injectable, PLATFORM_ID, Optional, isDevMode, Renderer2, RendererFactory2, ViewEncapsulation } from '@angular/core'
-import { Router } from '@angular/router'
+import { filter } from 'rxjs/internal/operators/filter'
```

Comment:
this import doesn't need to be from `internal`, but otherwise, lgtm
```suggestion
import { filter } from 'rxjs/operators'
```

Reply:
Good spot, thank you.